spec/requests/pages_spec.rb

```diff
@@ -3,6 +3,34 @@
 describe "REST" do
   let(:page) { Page.create(title: 'Foo', slug: 'foo-bar') }
 
+  describe 'MessageHandler' do
+    describe 'update_pages' do
+      context "page does not exist on ActionKit" do
+        let(:params) do
+          {
+            type: 'update_pages',
+            petition_uri:  "",
+            donation_uri:  "",
+            params: {
+              tags: ["/rest/v1/tag/5678/"]
+            }
+          }
+        end
+
+        before do
+          VCR.use_cassette('page_update_404') do
+            post "/message", params
+          end
+        end
+
+        it 'fails but is 200' do
+          expect(response.body).to match(/Sorry, this request could not be processed/)
+          expect(response.status).to eq(200)
+        end
```

Comment:
@Tuuleh I think then I should return 5xx so the job eventually ends up on the deadletter queue for inspection. And we should stop jobs we know will never run from being posted to the queue in the first place. wdt?